### ISSUE TITLE
extension can't exclude folders needed by other plugins

### ISSUE THREAD

**Issue by alvint** [JonathanWolfe/file-tree-exclude#1]
It's hardly fair to log the first bug report on this issue, but it must be documented that the method used here can't exclude folders explicitly used by other plugins.  For example, the `.git` folder will always be present/searched if either of the git extensions are used. This is a known issue with this approach and one that can't be addressed without upgrading to the next iteration of Brackets and a substantial rewrite.

**Comment by JonathanWolfe** [comment]
Yeah, I've noticed that and I'm still looking into ways of making this work with those folders as well. 

For the moment it just empties those folders from the filesystem so you still get some of the benefits.
